indico/util/mdx_latex_test.py

```diff
@@ -0,0 +1,29 @@
+__author__ = 'mic4ael'
```

Comment:
We don't use `__author__` (everyones owns everything here), and the license header is missing :wink: